mindjunkies/accounts/management/commands/populate_db.py

```
@@ -0,0 +1,72 @@
+            for student in enrolled_students:
+                Enrollment.objects.create(student=student, course=course, status="active")
+                if random.choice([True, False]):
+                    Rating.objects.create(
+                        course=course,
+                        student=student,
+                        rating=random.randint(1, 5),
+                        comment=fake.sentence()
+                    )
+            self.stdout.write(f"Created course: {course.title} with {num_enrollments} enrollments")
+
+        self.stdout.write(self.style.SUCCESS("Database population completed successfully!"))
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)